BrightID/src/components/Connections/ConnectionScreen.tsx

```diff
@@ -58,14 +64,17 @@ function ConnectionScreen(props: Props) {
     connection,
     verificationsTexts,
     connectedAt,
-    mutualGroups,
     mutualConnections,
+    mutualGroups,
     loading,
   } = props;
   const navigation = useNavigation();
+  const myConnections = useSelector(selectAllConnections);
 
   const [groupsCollapsed, setGroupsCollapsed] = useState(true);
   const [connectionsCollapsed, setConnectionsCollapsed] = useState(true);
+  const [possibleDuplicatesCollapsed, setPossibleDuplicatesCollapsed] = useState(true);
+  const possibleDuplicates = myConnections.filter(conn => stringSimilarity(conn.name, connection.name) >= 0.6 && conn.id != connection.id);
```

Comment:
It's better to add a constant to `@/utils/constants` for this `0.6`.

Reply:
Done